dataplane/linux/bpf_ep_mgr.go

```diff
@@ -485,22 +506,40 @@ func (m *bpfEndpointManager) applyProgramsToDirtyWorkloadEndpoints() {
 		err := errs[wlID]
 		if err == nil {
 			log.WithField("id", wlID).Info("Applied policy to workload")
+			if m.allWEPs[wlID] != nil {
+				if _, ok := m.happyWEPs[wlID]; !ok {
```

Comment:
nit: A bit confused by the two types of testing existence. Thought for a while that `nil` might be a valid happy value ...

Reply:
Agreed, not ideal.  I'm about to pout up another PR so will address under that one.